packages/node_modules/@ciscospark/plugin-people/src/people.js

```diff
@@ -142,7 +144,8 @@ const People = SparkPlugin.extend({
    *   });
    *   // => success
    */
-  list(options) {
+  list(options, showAllTypes = false) {
+    this.batcher.showAllTypes = showAllTypes;
```

Comment:
If list gets called twice with showAllTypes = true and showAllTypes = false, depending on timing, you could get unexpected behavior in the response.

Reply:
That is true; I was trying to avoid having to extend the "query" parameter, and thought this edge case wasn't a big deal.

I'll try to think of a simple way to make this work on a per request basis without a race condition, and without having to extend request.